source/locale/pt_BR/LC_MESSAGES/flow.po

```diff
@@ -1174,45 +1276,58 @@ msgid ""
 "`payload.metadata.timestamp`. See more in "
 ":numref:`using_cum_sum_node_cumulative` ."
 msgstr ""
-"No nó `cumulative sum`, nós iremos acumular o valor de `rain` (*Target attribute*) na janela de  tempo de 60 minutos (*Time period*) e iremos setar essa soma em um novo atributo chamado `payload.data.attrs.rain60Min` (*Sum*).  A configuração *Timestamp* refere-se ao  timestamp advindo do device ou da dojot. Na maioria das vezes pode ser setado com `payload.metadata.timestamp`. Veja mais em :numref:`using_cum_sum_node_cumulative`"
+"No nó `cumulative sum`, nós iremos acumular o valor de `rain` (*Target "
+"attribute*) na janela de  tempo de 60 minutos (*Time period*) e iremos "
+"setar essa soma em um novo atributo chamado "
+"`payload.data.attrs.rain60Min` (*Sum*).  A configuração *Timestamp* "
+"refere-se ao  timestamp advindo do device ou da dojot. Na maioria das "
+"vezes pode ser setado com `payload.metadata.timestamp`. Veja mais em "
+":numref:`using_cum_sum_node_cumulative`"
 
-#: ../../source/flow.rst:907
+#: ../../source/flow.rst:966
 msgid ": Cumulative sum node configuration"
 msgstr ": Configuração de Soma acumulada"
 
-#: ../../source/flow.rst:909
+#: ../../source/flow.rst:968
 msgid ""
 "We want the notification to be triggered only if the accumulated rain "
 "value is greater than 100, for that we will use the switch node. As in "
 "image :numref:`using_cum_sum_node_switch`."
-msgstr "Nós queremos que a notificação seja disparada apenas se o valor de chuva acumulado seja maior que 100, para isso usaremos o nó switch. Como na imagem :numref:`using_cum_sum_node_switch`."
+msgstr ""
+"Nós queremos que a notificação seja disparada apenas se o valor de chuva "
+"acumulado seja maior que 100, para isso usaremos o nó switch. Como na "
```

Comment:
I think it would look better visually if the `switch` node was formatted as code or italic.